request/request.go

```diff
@@ -98,6 +98,8 @@ func DapiRequest(body []byte, action string, header http.Header) ([]byte, error)
 	}
 
 	request.Header.Set("Content-Type", "application/json")
+	request.Header.Add("X-Dapi-Library-Version", goExtraHeaders.LibraryVersion)
```

Comment:
Use `Set` instead of `Add`.

Reply:
Fixed: https://github.com/dapi-co/dapi-go/pull/10/commits/dc28b61f8a32736443c4e747ec97084136d7d111